### PR TITLE
[CLOUD-3860] Initial version of XP 2.0 image

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.redhat.com/browse/CLOUD-3860
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by luck3y** [comment]
Note this should be held until we have final artifacts.

**Review by dbkreling — DISMISSED** [review]
LGTM

**Review by jfdenise — DISMISSED** [review]

File: modules/eap-xp-20-env/2.0.0/module.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      value: "/opt/eap/standalone/deployments"
+    - name: "com.redhat.dev-mode"
+      value: "DEBUG:true"
+      description: "Environment variable used to enable development mode (debugging). A value of true will enable development mode."
+    - name: "com.redhat.dev-mode.port"
+      value: "DEBUG_PORT:8787"
+      description: "Environment variable used to specify the debug port. If not set, the default EAP debug port will be used (8787). Only applicable when development mode is enabled."
+envs:
+    - name: "WILDFLY_VERSION"
+      value: "7.3.4.GA-redhat-00003"
```
Comment (dbkreling) [inline modules/eap-xp-20-env/2.0.0/module.yaml]: Shouldn't the version for XP2 should be `2.0.0.GA-redhat-00002` according to https://code.engineering.redhat.com/gerrit/gitweb?p=jbossas/jboss-eap7.git;a=blob;f=pom.xml;h=6fa374f58f64513a0221b077e319dac3903337ee;hb=be516742047a4ed0883fe08f7d46d07c296f3b9b#l38 ?

**Review by dbkreling — COMMENTED** [review]
Update: the final artifacts were promoted and can be found at http://download.devel.redhat.com/devel/candidates/middleware/eap-xp/JBEAPXP-2.0.0-GA-CR1/

File: modules/eap-xp-20-env/2.0.0/module.yaml (hunk 1 above)
Comment (jfdenise) [reply modules/eap-xp-20-env/2.0.0/module.yaml]: @dbkreling , yes 2.0.0.

Comment (luck3y) [reply modules/eap-xp-20-env/2.0.0/module.yaml]: Thanks, updated.

**Review by dbkreling — APPROVED** [review]

**Pull request merged**